www/_template/tutorials/svelte.md

```diff
@@ -0,0 +1,327 @@
+
+## Customize your project layout
+
+Snowpack is flexible enough to support whatever project layout that you prefer. In this guide, you'll learn how to use a popular project pattern from the Svelte community.
+
+```
+📁 src : your Svelte components and their assets (CSS, images)
+    ↳ index.js
+    ↳ App.svelte
+📁 public : global assets like images, fonts, icons, and global CSS
+    ↳ index.css
+    ↳ index.html
+```
+
+Use your favorite visual editor to rearrange and rename, or run these commands in the terminal:
+
+```bash
+mkdir src
+mkdir public
+mv index.js src/index.js
+mv App.svelte src/App.svelte
+mv index.html public/index.html
+mv index.css public/index.css
+```
+
+This means if you are running Snowpack right now, the site is now broken as the files are all in different places. Lets add a "mount" configuration to update your site to your new project layout.
+
+The `mount` configuration changes where Snowpack scan for and builds files. Head back to the `snowpack.config.js` file you edited when you added `@snowpack/plugin-svelte`. Add this to the empty `mount` object:
+
+```diff
+// snowpack.config.js
+
+  mount: {
+-   /* ... */
++   // directory name: 'build directory'
++   public: '/',
++   src: '/_dist_',
+  },
+```
+
+<img src="/img/guides/folder-structure.png" alt="Graphic shows the original and new folder structures side by side. Arrows indicate that the files are built to where the arrow points. The Original side shows a folder labeled ./ entire directory with an arrow pointing to a folder labeled  mysite.com/*. The New side shows a folder labeled ./src/* with an arrow pointing to a folder labeled mysite.com/_dist/*. Then a second folder labeled ./public/* with an arrow pointing to a folder labeled mysite.com/* " />
+
+`mount` is part of the [Snowpack Configuration API](https://www.snowpack.dev/#configuration). It allows you to customize the file structure of your project. The key is the name of the directory and the value is where you'd like them in the final build. With this new configuration, Snowpack builds files in `public` like `public/index.css` directory into `index.css`. It builds files in `src` like `src/index.js` into `/_dist_/index.js`, so change that path in your `index.html`:
+
+```diff
+<!-- public/index.html -->
+
+  <body>
+-   <h1>Welcome to Snowpack!</h1>
+-   <script type="module" src="/index.js"></script>
++   <script type="module" src="/_dist_/index.js"></script>
+  </body>
+```
+
+You'll need to restart Snowpack (stop the process in terminal and then run `npm start` again) for configuration file changes. When you start up again, if it worked, it should look exactly the same since all the changes you just made are behind the scenes.
```

Comment:
"it should look exactly the same since all the changes you just made are behind the scenes." maybe instead something a bit more positive, like:

"it should look exactly as it did before, but now using your brand new project folder layout."